.github/workflows/ci.yml

```diff
@@ -117,9 +117,42 @@ jobs:
         with: { version: "${{ env.protoc_version }}", repo-token: "${{ github.token }}" }
       - uses: dart-lang/setup-dart@v1
       - run: dart pub get
-      - run: dart pub run grinder protobuf
+      - run: dart run grinder protobuf
       - name: Deploy
-        run: dart pub run grinder pkg-github-release pkg-github-linux
+        run: dart run grinder pkg-github-release pkg-github-linux-ia32 pkg-github-linux-x64
+        env: {GH_BEARER_TOKEN: "${{ github.token }}"}
+
+  deploy_github_linux_qemu:
+    name: "Deploy Github: Linux"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          # https://github.com/dart-lang/sdk/pull/48665
+          # - arch: arm
+          #   platform: linux/arm/v7
+          - arch: arm64
+            platform: linux/arm64
+    needs: [deploy_github_linux]
+    if: "startsWith(github.ref, 'refs/tags/') && github.repository == 'sass/dart-sass-embedded'"
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: arduino/setup-protoc@v1
+        with: { version: "${{ env.protoc_version }}", repo-token: "${{ github.token }}" }
+      - uses: dart-lang/setup-dart@v1
+      - run: dart pub get
+      - run: dart run grinder protobuf
```

Comment:
Will the protobuf files generated by these tasks be visible inside the Docker container, or should we run the protobuf compiler in there instead?

Reply:
The generated file is inside workspace, so that it will be available in container. It is easier this way rather than installing protoc inside the container.

Reply:
Here is an example of using this method to run pkg-standalone-linux-arm64:

https://github.com/ntkme/dart-sass-embedded/runs/5700309782?check_suite_focus=true